community/bonita-distrib/bonita-admin-application.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<applications xmlns="http://documentation.bonitasoft.com/application-xml-schema/1.0">
+    <application token="adminApp" version="2.0" profile="Administrator" homePage="admin-process-list" state="ACTIVATED" layout="custompage_layoutBonita" theme="custompage_themeBonita">
+        <displayName>Bonita Administrator Application</displayName>
+        <iconPath>bonita-admin-application.png</iconPath>
+        <applicationPages>
+            <applicationPage customPage="custompage_adminUserDetailsBonita" token="admin-user-details"/>
```

Comment:
* I guess both `bonita-admin-application(-sp).xml` in `community/applications` and `subscription/applications` can be removed?
* Also, what's the impact of removing the `vX` at the end of the `customPage` token of each `applicationPage`?

Reply:
not really.
- the xml in the applications dir that contain the versions of the pages are needed to build the .bos
- the xml in bonita-distrib does not contain the versions because the pages we deploy in the distrib do not contain the versions (their page.properties in the zip of the pages override the UID page name to remove it)

the reason for this is that the pages without version in their token are migrated automatically when there is a new version of the pages. and the pages imported with the .bos contain the version so that they do not override the pages we provide in the distrib

Reply:
so we need both xml unfortunately

Reply:
mmm ok I see.
It still seems to be error-prone to have this duplication, but it's good enough for now. Maybe next time we need to change things there we can review that.
Thanks for the explanation 👍